string myName
string name
string type
string description
